### PR TITLE
Update: multiple sdk issue

### PR DESCRIPTION
**Pull request by dpaez**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- multiple sdk issue
```
Comment (juliangruber) [inline CHANGELOG.md:9]: it's not really a multiple sdk issue, it's a loading issue right?

File: index.js [hunk 2]
```diff
@@ -413,20 +414,25 @@ class SDK {
...
-          // Note(dk): consider add a timeout
-          await once(driveWatch, 'put-end')
         }
 
+        let unwatch
+        await new Promise(resolve => {
+          unwatch = drive.watch('', () => {
+            return resolve()
+          })
+        })
```
Comment (juliangruber) [inline index.js:433]:
```suggestion
        const unwatch = await new Promise(resolve => {
          let unwatch = drive.watch('', () => {
            resolve(unwatch)
          })
        })
```

Comment (juliangruber) [reply index.js:433]: Same below. This seems more direct to me

File: tests/childProcess2.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+  const commons = new P2PCommons({
+    disableSwarm: true,
+    persist: true,
+    baseDir: dir,
+    watch: true
+  })
+
+  await commons.ready()
+
```
Comment (juliangruber) [inline tests/childProcess2.js:17]:
```suggestion
```

isn't this implicit now?

File: tests/childProcess2.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  await commons.init({
+    type: 'profile',
+    title: 'title'
+  })
+
+  await commons.destroy()
+  process.exit(0)
```
Comment (juliangruber) [inline tests/childProcess2.js:24]:
```suggestion
```

Why do we need this? Shouldn't the process clean up by itself? Or is this for perf?

File: tests/index.js [hunk 5]
```diff
@@ -762,6 +757,14 @@ test('register - local contents', async t => {
     join(p2p.baseDir, encode(content1.url), 'file.txt'),
     'hola mundo'
   )
+
+  await new Promise(resolve => {
```
Comment (juliangruber) [inline tests/index.js:761]: core has https://nodejs.org/api/events.html#events_events_once_emitter_name for this case

Comment (juliangruber) [reply tests/index.js:761]:
```js
const src = await once(driveWatch, 'put-end')
```

File: CHANGELOG.md (hunk 1 above)
Comment (dpaez) [reply CHANGELOG.md:9]: oh yes, I was a bit confused on how to naming this. I'm going to use the loading issue. thanks!

File: tests/childProcess2.js (hunk 4 above)
Comment (dpaez) [reply tests/childProcess2.js:24]: If I remove that, the spawned process doesn't end. Seems odd. `destroy` ends OK, though.

File: index.js (hunk 2 above)
Comment (dpaez) [reply index.js:433]: cool, i like it

**Comment by dpaez** [comment]
Suggestions applied

File: tests/childProcess2.js (hunk 4 above)
Comment (juliangruber) [reply tests/childProcess2.js:22]: does it not end the first or second time?

File: index.js [hunk 6]
```diff
@@ -413,20 +414,24 @@ class SDK {
...
-          // Note(dk): consider add a timeout
-          await once(driveWatch, 'put-end')
         }
 
+        const unwatch = await new Promise(resolve => {
+          let unwatch = drive.watch('', () => {
+            resolve(unwatch)
+          })
+        })
+        if (unwatch) unwatch.destroy()
```
Comment (juliangruber) [inline index.js:433]: when would this not exist?

File: tests/childProcess2.js (hunk 4 above)
Comment (dpaez) [reply tests/childProcess2.js:22]: the second time 🤔

File: index.js (hunk 6 above)
Comment (dpaez) [reply index.js]: removed! 😅

File: tests/childProcess2.js (hunk 4 above)
Comment (juliangruber) [reply tests/childProcess2.js:22]: hm that's a bug that could affect the CLI probably

Comment (dpaez) [reply tests/childProcess2.js]: removed 🎉

File: tests/childProcess2.js (hunk 3 above)
Comment (dpaez) [reply tests/childProcess2.js]: yeah, removed

**Pull request merged**